corrected the grammatical mistakes (#58)
anitab-org/documentation: README.md

## Contributing

> We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following:-

Reviewer: ```suggestion
We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following:
```

> We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following lists.

Reviewer 2: ```suggestion
We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend the following :-
```

  - Join our [Zulip](http://anitab-org.zulipchat.com) community to read our discussions and be part of them! Learn more about how we use Zulip [here](/zulip.md).
  - Attend our Open Community sessions to interact with community members and ask questions. You can check our calendars here: [PST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles) || [EST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/New_York) || [GMT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=GMT) || [WAT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Africa/Lagos) || [IST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Asia/Colombo) || [AEDT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Australia/Sydney)

## Documentation

> Below is a list of documents currently in this repository. If none of these documentations helps, feel free to reach out to us either by asking your question on Zulip or creating an issue requesting the information you need.

Reviewer 2: ```suggestion
Below is a list of documents currently in this repository. If none of these documents help, feel free to reach out to us either by asking your question on Zulip or creating an issue requesting the information you need.
```
